packages/website/pages/docs/guides/run-a-node.mdx

```diff
@@ -19,10 +19,31 @@ This guide will walk you through the process of operating a Taiko node via [simp
 
 <Steps>
 ### Clone simple-taiko-node
-```sh
-git clone https://github.com/taikoxyz/simple-taiko-node.git
-cd simple-taiko-node
-```
+
+{" "}
+
+<Tabs items={["Mac", "Linux", "Windows"]} defaultIndex="0">
+  <Tab>
+  ```sh
+  git clone https://github.com/taikoxyz/simple-taiko-node.git
+  cd simple-taiko-node
+  ```
+  </Tab>
+  <Tab>
+  ```sh
+  git clone https://github.com/taikoxyz/simple-taiko-node.git
+  cd simple-taiko-node
+  ```
+  </Tab>
+  <Tab>
+  ```sh
+  git config --global core.autocrlf false.
```

Comment:
```suggestion
  git config --global core.autocrlf false
```
i think we need this right?

Reply:
I apologize for the mistake. Thank you for making the correction.